build.zig.zon

```diff
@@ -25,6 +25,10 @@
             .hash = "1220af0752814ec0a3438aa3ca3e33661885ec134c5e7b794247381b0fbe90c21012",
             .lazy = true,
         },
+        .zsdl = .{
+            .path = "https://github.com/zig-gamedev/zsdl/archive/9c87bd98451e21a98756e7120b279f9f7f3ac1fc.tar.gz",
```

Comment:
This needs to be pointed to a valid commit containing https://github.com/zig-gamedev/zsdl/pull/7/commits/9c87bd98451e21a98756e7120b279f9f7f3ac1fc and also needs `url` and `hash` instead of `path`.